crates/update/Cargo.toml

```diff
@@ -6,6 +6,10 @@ rust-version.workspace = true
 license-file = "LICENSE"
 publish = false
 
+[features]
+# NOTE(bfops): This is not a well-thought-through feature. It's really only meant for internal testing/debugging.
```

Comment:
I would maybe clarify here that this is for CI

Reply:
done